dora/tests/testcontainers/src/test/java/alluxio/membership/MembershipManagerTest.java

```diff
@@ -191,6 +191,11 @@ public void testEtcdMembershipWithoutAuth() throws Exception {
     testEtcdMembership(getHealthyEtcdMemberMgr());
   }
 
+  @Test
+  public void testDefaultEtcdMembershipWithoutAuth() throws Exception {
```

Comment:
serviceregistry

Reply:
whats the difference with testEtcdMembershipWithoutAuth?
think u want to do a getHealthyServiceRegistryMemberMgr instead inside of this?

Reply:
removed